mautic/mautic
# Css fixes notes

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8490
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
View any detail page that has a "recent activity" timeline and you'll see that the details no longer align to the right of the bullets.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:

1. Go to a email details page (or any other details page such as forms; create an entity if it doesn't exist)
2. Look to the right notice the timeline UI is off compared to M2
3. Also go to a contact details page, click on the notes tab and create a new note. The notes grid does not look OK.

#### Steps to test this PR:
0. Load up [this PR](https://mautibox.com)
1. Go to a email details page (or any other details page such as forms; create an entity if it doesn't exist)
2. Look to the right notice the timeline is OK
3. Also go to a contact details page, click on the notes tab and create a new note. Notice that the grid with notes looks OK.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
